src/common/components/textWithLineBreaks/__tests__/TextWithLineBreaks.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { shallow } from 'enzyme';
+import * as React from 'react';
```

Comment:
why import * as React from 'react'?

Reply:
eslint is showing following error "React must be in scope when using JSX" if React is not imported